rqt_shell/src/rqt_shell/shell.py

```diff
@@ -85,9 +86,16 @@ def __init__(self, context):
         super(Shell, self).__init__(context)
         self._context = context
         self.setObjectName('Shell')
+        self.args = self._parse_args(context.argv())
 
         self._widget = None
 
+    def _parse_args(self, argv):
+        parser = argparse.ArgumentParser(prog='rqt_shell', add_help=False)
```

Comment:
However, ```add_help=False``` is present [here](https://github.com/ros-visualization/rqt_common_plugins/blob/2c2487a02c83628a14be7bde318d20249167ffdb/rqt_plot/src/rqt_plot/plot.py#L68)